nolearn/lasagne/visualize.py

```diff
@@ -338,29 +347,31 @@ def make_pydot_graph(layers, output_shape=True, verbose=False):
     return pydot_graph
 
 
-def draw_to_file(layers, filename, **kwargs):
+def draw_to_file(net, filename, **kwargs):
```

Comment:
I'm also not sure why we would need backwards compatibility here.

Anyway, if both modes were supported, could you please adjust the tests to test both?